coralproject/talk
# Remove duplicate placeholder proptype.

## What does this PR do?
Remove a duplicated PropType `placeholder` from Editor component. 

Related to issue #2034 

## How do I test this PR?

- Enable talk-plugin-rich-text
- Run Talk
- Check that rich text is enabled on Stream
- Make a comment with rich text
